lms/djangoapps/api_manager/users/views.py

```diff
@@ -143,7 +143,10 @@ def _manage_role(course_descriptor, user, role, action):
     Helper method for managing course/forum roles
     """
     supported_roles = ('instructor', 'staff', 'observer', 'assistant')
-    forum_moderator_roles = ('instructor', 'staff', 'assistant')
+    role_perm_map = {
+        FORUM_ROLE_MODERATOR: ('instructor', 'staff'),
+        FORUM_ROLE_COMMUNITY_TA: ('assistant',),
```

Comment:
What does the `FORUM_ROLE_COMMUNITY_TA` role grants as privileges, exactly? On this client's instance, TAs *are* moderators, so I'd like to make sure we don't remove some of their moderation capabilities with this change. 

Also, the end goal is to ensure the internal admins don't get flagged as TAs, but we do want to keep the existing behavior and labels for course staff. From what I see in apros, both staff and TAs get the same label in apros (`/static/scss/lesson-content/discussion.scss#L323-L340`).

Given that the end goal here is only to ensure that internal admins don't get flagged as TAs, why not use the `FORUM_ROLE_ADMINISTRATOR` for them (cf https://github.com/edx-solutions/edx-platform/blob/4ae4f7c919ec2d20c3ddca0854d29cb0d941aee7/lms/djangoapps/instructor/access.py#L84 ), and keep the current behavior for instructor/staff/assistant? That would likely give you another label (I'm guessing "Administrator"), which would likely be fine to show in the forums for Internal admins.